Lib/typing.py

```diff
@@ -116,7 +116,7 @@
 # The pseudo-submodules 're' and 'io' are part of the public
 # namespace, but excluded from __all__ because they might stomp on
 # legitimate imports of those modules.
-
+# Union = type(int | str)
```

Comment:
Why is this back (albeit commented out :-)?

Reply:
Oof. Merge issue, I suspect. Thanks for catching it :)